### ISSUE TITLE
Blog index - Layout improvements

### ISSUE THREAD

**Issue by Guillaumegranger1** [Shopify/dawn#418]
TL;DR The `Blog index` (and blog post section) doesn't provide sufficient flexibility for the featured images to appear in an acceptable size. I'm proposing to add some settings to better customize the image height, alongside a toggle to enable a grid view without full-width cards.

__________

![image](https://user-images.githubusercontent.com/15196462/129753156-ef98be9b-f378-4f03-8257-dbc5dd14a342.png)

1. When the layout is set to `Grid` instead of `Collage` , no cards are displayed full-width on the desktop and layout view. If the number of cards is uneven, we leave some white space as a preferred tradeoff over huge images:

![image](https://user-images.githubusercontent.com/15196462/129750663-aa5ebf6d-9e40-4fd8-99e3-58f231103175.png)
 
2. The `Featured image height` setting has different values for the `Small-Medium-Large` options depending on whether the card is taking 2 columns or one:

![image](https://user-images.githubusercontent.com/15196462/129751824-771b2735-17b9-4c22-b760-b749879c5531.png)

3. Finally, I think we should do something similar with the `Blog post section` which currently displays an enormous image when only one article exist in the blog. My proposition would be not to add any setting, but just to default to 2 columns when there's 2 or less articles from the blog, and leave some white space for the single article use case:

Current layout when there's 1 article in the featured blog:
![image](https://user-images.githubusercontent.com/15196462/129752610-b1e3416d-2a72-43d4-911c-d55bfa1f9ed8.png)

@nicklepine @Oliviammarcello @melissaperreault 

[Link to internal mockups](https://www.figma.com/file/JcYdswrk2JhqBTGBVWPM7J/Dawn-2.0?node-id=12271%3A67794)

**Comment by melissaperreault** [comment]
cc. 👀 @gretchen-willenberg to add on the radar of new settings to review/update, you can see the details in yellow on the screenshots and link to internal mockups at the bottom of the issue. 🙏

**Comment by gretchen-willenberg** [comment]
Thanks! Added to the copy doc. No changes to the labels or text, but a few questions on aligning order of the drop down choices and removal of a link to help doc on image aspect ratio. 

[Blog template](https://docs.google.com/document/d/14Aa-twJrAWSo95v7RwTF0M2Pk5xJE3t10GJfmcesWBU/edit#bookmark=id.llhyj9wcivk8)
- Made revisions/additions
- Question on `aspect ratio` copy and removal of learn more link
- Update `change excerpts` copy to align with final text 

[Blog posts section](https://docs.google.com/document/d/14Aa-twJrAWSo95v7RwTF0M2Pk5xJE3t10GJfmcesWBU/edit#bookmark=id.8owqi1fes70d)
- Made revisions/additions
- Question on `aspect ratio` copy and removal of learn more link
- Question if `change excerpts` copy should also be added to Title block to align with revisions

**Comment by melissaperreault** [comment]
> Question on aspect ratio copy and removal of learn more link

`Learn more` link should be there, this was overlooked!
